schemas/eventLog.json

```diff
@@ -0,0 +1,245 @@
+{
+	"title": "Event Log",
+	"description": "Logged action taken regarding the submission.",
+	"required": [
+		"assocType",
+		"assocId",
+		"dateLogged"
+	],
+	"properties": {
+		"assocId": {
+			"type": "integer",
+			"description": "The submission or submission file ID.",
+			"apiSummary": true
+		},
+		"assocType": {
+			"type": "integer",
+			"description": "The assoc object. This should always be `ASSOC_TYPE_SUBMISSION` or `ASSOC_TYPE_SUBMISSION_FILE`.",
+			"apiSummary": true
+		},
+		"copyrightNotice": {
+			"type": "string",
+			"description": "The context derived copyright notice confirmed for the submission",
+			"multilingual": true,
+			"validation": [
+				"nullable"
+			]
+		},
+		"dateLogged": {
+			"type": "string",
+			"description": "The date this event was recorded.",
+			"apiSummary": true,
+			"writeDisabledInApi": true,
+			"validation": [
+				"date_format:Y-m-d H:i:s"
+			]
+		},
+		"decision": {
+			"type": "integer",
+			"description": "The decision type identifier. One of the Decision::* constants.",
+			"validation": [
+				"nullable",
+				"min:1",
+				"max:32"
```

Comment:
This might cause problems, for example, if a third-party plugin adds a decision and gives it a constant `555`.